firebolt_provider/example_dags/example_firebolt.py

```diff
@@ -53,14 +53,10 @@
     default_args={"firebolt_conn_id": FIREBOLT_CONN_ID},
     tags=["example"],
     catchup=False,
+    template_searchpath=[PATH_TO_TEMPLATES],
```

Comment:
Is this also a part of that reverted change with .sql files? Let's clean it up as much as possible.

Reply:
Will do